libs/ledger/include/ledger/chain/block_coordinator.hpp

```diff
@@ -399,6 +404,7 @@ void BlockCoordinator::SetBlockPeriod(std::chrono::duration<R, P> const &period)
 inline void BlockCoordinator::EnableMining(bool enable)
 {
   mining_enabled_ = enable;
+  syncronised_    = enable;
```

Comment:
It is not obvious why this flag is set by `enable`. Does mining enable automatically imply that constellation is synchronised?

Reply:
yes it does